### PR TITLE
Show a balance for the Token To, update position of the MM fee, removes a link

### PR DESCRIPTION
**Pull request by dan437**
## Explanation
- Shows balance for the Token To
- Updates position of the MM fee and adds a link to view all quotes
- Removes a link from the Quote rate

## Screenshots
image

## Testing
- On the homepage click on the "Swap" button
- Select Token From, Token To where you have some balance and amount
- See a balance for Token To under its symbol, no link in the Quote rate line and a new line with the MM fee info + link to view all quotes

### CODE REVIEW

**Comment by brad-decker** [comment]
E2E test failures seem related @dan437

File: ui/pages/swaps/loading-swaps-quotes/__snapshots__/loading-swaps-quotes-stories-metadata.test.js.snap [hunk 1]
```diff
@@ -27,4 +27,4 @@ exports[`storiesMetadata matches expected values for storiesMetadata 1`] = `
...
   },
 }
-`;
\ No newline at end of file
+`;
```
Comment (brad-decker) [inline ui/pages/swaps/loading-swaps-quotes/__snapshots__/loading-swaps-quotes-stories-metadata.test.js.snap:30]: This change seems unintentional, and should likely be removed from the PR

File: ui/pages/swaps/exchange-rate-display/index.scss [hunk 2]
```diff
@@ -18,5 +18,11 @@
 
   &__quote-rate {
     cursor: pointer;
+    color: var(--color-primary-default);
+  }
+
+  &__quote-rate--no-link {
+    cursor: text;
+    color: var(--color-text-default);
```
Comment (brad-decker) [inline ui/pages/swaps/exchange-rate-display/index.scss:26]: Why use SCSS colors versus the color prop on Typography or Box?

File: ui/pages/swaps/prepare-swap-page/review-quote.test.js [hunk 3]
```diff
@@ -93,7 +95,8 @@ describe('ReviewQuote', () => {
     const { getByText } = renderWithProvider(<ReviewQuote {...props} />, store);
     expect(getByText('New quotes in')).toBeInTheDocument();
     expect(getByText('Quote rate')).toBeInTheDocument();
-    expect(getByText('MetaMask fee')).toBeInTheDocument();
+    expect(getByText('Includes a 1% MetaMask fee –')).toBeInTheDocument();
+    expect(getByText('view all quotes')).toBeInTheDocument();
```
Comment (brad-decker) [inline ui/pages/swaps/prepare-swap-page/review-quote.test.js:99]: Does this constitute a change in our fee or just moving it around?

**Review by brad-decker — CHANGES_REQUESTED** [review]

File: ui/pages/swaps/prepare-swap-page/review-quote.test.js (hunk 3 above)
Comment (dan437) [reply ui/pages/swaps/prepare-swap-page/review-quote.test.js:99]: No change in the fee logic, just making it clearer that our MM fee is included in quotes already.

File: ui/pages/swaps/loading-swaps-quotes/__snapshots__/loading-swaps-quotes-stories-metadata.test.js.snap (hunk 1 above)
Comment (dan437) [reply ui/pages/swaps/loading-swaps-quotes/__snapshots__/loading-swaps-quotes-stories-metadata.test.js.snap:30]: I've updated snapshots and this is what I got. When I run `yarn jest --updateSnapshot` now, it doesn't give me anything new.

**Review by brad-decker — APPROVED** [review]

**Review by darkwing — APPROVED** [review]

**Pull request merged**